source/includes/_security.md

```diff
@@ -1,5 +1,132 @@
+
+## FHIR Resource Access
+
+The CDS Service is able to use the FHIR server of the EHR to obtain any additional data it needs in order to perform its decision support. This is similar to SMART on FHIR where the SMART app can obtain additional data via the provided FHIR server.
+
+Like SMART on FHIR, CDS Hooks requires that access to the FHIR server be controlled by an Authorization server utilizing the OAuth 2 framework. Thus, the CDS Service is able to consume the given FHIR server via an access (bearer) token just like a SMART app. While CDS Hooks shares the underlying technical framework and standards as SMART on FHIR, there are very important differences between SMART and CDS Hooks.
+
+### Obtaining an Access Token
+
+In SMART on FHIR, the SMART app requests and ultimately obtains an access token from the Authorization server using the SMART launch workflow. This process utilizes the authorization code grant model as defined by the OAuth 2.0 Authorization Framework in [rfc6749](https://tools.ietf.org/html/rfc6749).
+
+With CDS Hooks, the EHR provides the access token directly in the request to the CDS Service. Thus, the CDS Service does not need to request the token from the authorization server as a SMART app would. This is done purely for performance reasons as the authorization code grant model in OAuth 2 involves several HTTPS calls and redirects. In contrast with a SMART app, a CDS Service may be invoked many times during a workflow. Going through the authorization code grant model on every hook invocation would likely result in a slow performing CDS Service due to the authorization overhead.
+
+```json
+{
+  "fhirAuthorization" : {
+    "access_token" : "some-opaque-fhir-access-token",
+    "token_type" : "code",
+    "expires_in" : 300,
+    "scope" : "patient/Patient.read patient/Observation.read"
+  }
+}
+```
+
+### Access Token
+
+The access token is specified in the CDS Service request via the `fhirAuthorization` request parameter. This parameter is an object that contains both the access token as well as other related information.
+
+Field | Description
+----- | -----------
+`access_token` |*string*. This is the OAuth 2 access token that provides access to the FHIR server.
+`token_type`   |*string*. Always the value `code`
+`expires_in`   |*integer*. The lifetime in seconds of the access token.
+`scope`        |*string*. The scopes the access token grants the CDS Service.
+
+It is recommended that the `expires_in` value be very short lived as the access token should be treated as a transient value by the CDS Service.
```

Comment:
What timestamp is the `expires_in` value being used with to determine if it has expired?

Reply:
Normally, the developer (client role in OAuth 2 speak) would use this value to compare against the current date/time and know when they need to request a refresh token. (See [RFC 6749 section 4.2.2](https://tools.ietf.org/html/rfc6749#section-4.2.2) for more info on this field).

With CDS Hooks, since the usage of the FHIR access token is transient and the CDS Service is running for no more than a couple of seconds, the CDS Service developer shouldn't need to worry about this. I included it here for completeness as it is part of the access token response from the authorization server.

I'll add further documentation around this to make this more clear.